infrastructure/gonogo/gonogo.cc

```diff
@@ -24,11 +24,17 @@ void GoNoGo::go() {
 void GoNoGo::nogo(const std::string &status_message) {
     gonogomessage_.ready = false;
     gonogomessage_.status_message = status_message;
+
+    // If it's changing state
+    if (gonogomessage_.ready){
```

Comment:
This looks like it'll never run - it's right after the ready member is set to false. Shouldn't we instead keep track of the previous state and only print when it doesn't match the current state?